tracer/src/Datadog.Trace.Manual/Stubs/NullSpan.cs

```diff
@@ -45,6 +45,8 @@ public string ServiceName
 
     public ulong TraceId => Context.TraceId;
 
+    public string? RawTraceId => null;
```

Comment:
```suggestion
    public string? RawTraceId => Context.RawTraceId;
```

Reply:
Yeeeah.... nice try... you almost got me. That API doesn't exist in Datadog.Trace.Manual